<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Optimize2Widget</class>
 <widget class="QWidget" name="Optimize2Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>460</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>460</height>
    </rect>
   </property>
   <widget class="QWidget" name="top" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>920</width>
      <height>130</height>
     </rect>
    </property>
    <widget class="WaveBallWidget" name="waveBall" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
    </widget>
    <widget class="ProgressBarWidget" name="progreBar" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>126</y>
       <width>920</width>
       <height>4</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>401</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>扫描中</string>
     </property>
    </widget>
    <widget class="QLabel" name="subtitle">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>381</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>全面升级电脑开机速度,系统速度,上网速度,硬盘速度</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>50</y>
       <width>90</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>取消扫描</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="bottom" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>920</width>
      <height>331</height>
     </rect>
    </property>
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>921</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WaveBallWidget</class>
   <extends>QWidget</extends>
   <header location="global">customviews/waveballwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProgressBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">customviews/progressbarwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
